lib/ansible/modules/cloud/amazon/rds_instance.py

```diff
@@ -14,6 +14,9 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible. If not, see <http://www.gnu.org/licenses/>.
 
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
```

Comment:
I'm putting it in generally as a cut and paste understanding it should reduce compatibility problems by making more python 3 code behave the same as python 2 more of the time.